thrift/src/test/java/com/linecorp/armeria/it/thrift/ThrottlingRpcServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package com.linecorp.armeria.it.thrift;
+
+import static com.linecorp.armeria.common.thrift.ThriftSerializationFormats.BINARY;
+import static com.linecorp.armeria.server.throttling.ThrottlingStrategy.always;
+import static com.linecorp.armeria.server.throttling.ThrottlingStrategy.never;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import org.apache.thrift.TApplicationException;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.linecorp.armeria.client.ClientBuilder;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.thrift.THttpService;
+import com.linecorp.armeria.server.thrift.ThriftCallService;
+import com.linecorp.armeria.server.throttling.ThrottlingRpcService;
+import com.linecorp.armeria.service.test.thrift.main.HelloService;
+import com.linecorp.armeria.testing.server.ServerRule;
+
+public class ThrottlingRpcServiceTest {
```

Comment:
Any good place to put this test? Because I want to just RPC service in `ServerRule`.